hedera-node/hedera-app/src/main/java/com/hedera/node/app/tss/handlers/TssVoteHandler.java

```diff
@@ -92,41 +90,31 @@ public static boolean hasReachedThreshold(
         if (activeRoster == null) {
             throw new IllegalArgumentException("No active roster found");
         }
-
-        // Get all votes for the active roster
-        final Map<RosterEntry, TssVoteTransactionBody> voteByNode = new HashMap<>();
-
         // Get the target roster from the TssVoteTransactionBody
         final Bytes targetRosterHash = tssVoteTransaction.targetRosterHash();
 
         // Also get the total active roster weight
         long activeRosterTotalWeight = 0;
-
+        // Initialize a counter for the total weight of votes with the same vote byte array
+        long voteWeight = 0L;
         final var tssBaseStore = context.storeFactory().writableStore(WritableTssStore.class);
         // For every node in the active roster, check if there is a vote for the target roster hash
-        for (final RosterEntry rosterEntry : rosterStore.getActiveRoster().rosterEntries()) {
+        for (final RosterEntry rosterEntry : activeRoster.rosterEntries()) {
             activeRosterTotalWeight += rosterEntry.weight();
-            final TssVoteMapKey tssVoteMapKey = new TssVoteMapKey(targetRosterHash, rosterEntry.nodeId());
+            final var tssVoteMapKey = new TssVoteMapKey(targetRosterHash, rosterEntry.nodeId());
```

Comment:
extremely minor nit: wish we didn't use `var` as this makes the code difficult to read. It's a lot easier to read it when all the types are explicitly stated.

Reply:
In services we mostly use `var` . I agree it should be consistent all over the project. Worth a larger discussion. Will fix in the next PR. Thanks @anthony-swirldslabs